app/controllers/api/current/exotel_sessions_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Api::Current::ExotelSessionsController < ApplicationController
+  def passthru
+    session = ExotelSession.new(params[:From], parse_digits)
+
+    if session.save(params[:CallSid])
+      render plain: "OK", status: 200
```

Comment:
The more idiomatic way of doing this seems to be to use the [head](https://guides.rubyonrails.org/layouts_and_rendering.html#using-head-to-build-header-only-responses) method.

Reply:
Head appears to default to `text/html`. Probably `head` along with a default `:text` format in `routes.rb` is the cleanest way to go about.